tests/alignment_test.json

```diff
@@ -0,0 +1,16 @@
+{
+    "small_structs": [
+        {
+            "var_0": 2,
+            "var_1": 1
+        },
+        {
+            "var_0": 3,
+            "var_1": 1
+        },
+        {
+            "var_0": 4,
+            "var_1": 1
+        }
+    ]
+}
```

Comment:
this file wasn't used to generate the annotated `alignment_test_{before,after}_fix.afb` files because probably the `flatc --binary` doesn't use the functions that lead to the corrupt vector alignment (`CreateVectorOfStructs/StartVectorOfStructs`), however I haven't checked exactly the `flatc --binary` way. Though `CreateUninitializedVectorOfStructs` function is known to work correctly before/after fix, which maybe exaplains the correctness of `flatc --binary`. I can dive deeper into this if needed.

Reply:
no need